modules/openid_auth/app/controllers/openid_auth/docs/v0/validation_controller.rb

```diff
@@ -3,7 +3,7 @@
 module OpenidAuth
   module Docs
     module V0
-      class ValidationController < ApplicationController
+      class ValidationController < ::ApplicationController
         skip_before_action(:authenticate)
```

Comment:
won't you need a skip before action `validate_user` here since with docs we won't have the token and all that?

Reply:
You're right, I was trying to reference the base ApplicationController class the same way some of the other modules had done for docs but at the end I just moved the before action to the validation controller directly.